library/common-library/templates/_naming.tpl

```diff
@@ -26,7 +26,7 @@ If release name contains chart name it will be used as a full name.
 {{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" }}
 {{- else }}
 
-{{- $name := default .Chart.Name .Values.nameOverride }}
+{{- $name := default (include "common.naming.chartnameOverride" .) .Values.nameOverride }}
```

Comment:
Could we change this default to use the `|` so that it is obvious the order?

Reply:
Solved in [#5cd2662](https://github.com/newrelic/helm-charts/pull/758/commits/5cd26625c7c25f6ff377376eb450dbfbdb6bff6f)